caldav/objects.py

```diff
@@ -47,6 +47,29 @@ def errmsg(r):
     return "%s %s\n\n%s" % (r.status, r.reason, r.raw)
 
 
+def _expand_event(event, start, end):
+    """
+    :param event: Event
+    """
+    import icalendar
```

Comment:
I have added icalendar to the list of dependencies now, so "import icalendar" should be moved to the top of the file ... but of course it needs to be done consistently.  I'm surprised that I forgot doing it myself.